### PR TITLE
fix(manager): Adding steps copy feature on empty web property, URL Validation on publish domain URL 

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix
1) Allows users to copy the initial steps for empty web property.
2) URL validation on Publish domain on web property details page and env page.

## Does this PR introduce a breaking change

No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

1) Allows users to copy the initial steps for empty web property.
image

2) URL Validation on web property details page
image

3) URL Validation on create env page
image

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by hybridx** [comment]
Screenshot 2023-01-19 at 2 39 59 PM
Have we tried this @soumyadip007 ?

**Review by hybridx — APPROVED** [review]

**Review by soumyadip007 — APPROVED** [review]

File: packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx [hunk 1]
```diff
@@ -206,7 +206,11 @@ export const WebPropertyEnvPage = (): JSX.Element => {
                               target="_blank"
                               rel="noopener noreferrer"
                             >
-                              <ExternalLinkAltIcon /> {env.url}
+                              <ExternalLinkAltIcon />
+                              {/* {env.url} */}
```
Comment (akhilmhdh) [inline packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx:210]: Suggestion: if not u can remove this

File: packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ import { ActivityStream } from '@app/components/ActivityStream';
 import { Ephemeral } from './components/Ephemeral';
 import { EmptyInfo } from './components/EmptyInfo';
 
-const URL_LENGTH_LIMIT = 50;
+const URL_LENGTH_LIMIT = 30;
```
[the remark this answers is not in the document]
Comment (akhilmhdh) [reply packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx:54]: The variable is kept as easy to adjust for the page. It's a page specific entity thus exporting doesn't have purpose

**Review by akhilmhdh — APPROVED** [review]

File: packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx]: done

**Pull request merged**